pulsar4s-core/src/main/scala/com/sksamuel/pulsar4s/ConsumerMessage.scala

```diff
@@ -18,6 +18,8 @@ trait ConsumerMessage[T] {
 
   def value: T
 
+  def valueTry = Try(value)
```

Comment:
valueSafe ?

Reply:
I slightly prefer the `Try` naming since it's clear what it is. There are many reasons why something might be "safe" or "unsafe".